deploy/kustomize/kubernetes-distributed-lvm/kustomization.yaml

```diff
@@ -0,0 +1,65 @@
+bases:
+- ../kubernetes-base-lvm
+
+resources:
+- webhooks-rbac.yaml
+
+patchesJSON6902:
+- target:
+    group: apps
+    version: v1
+    kind: StatefulSet
+    name: pmem-csi-controller
+  patch: |-
+    - op: add
+      path: /spec/template/spec/containers/0/command/-
+      value: -mode=webhooks
+    - op: add
+      path: /spec/template/spec/containers/0/command/-
+      value: --schedulerListen=:8000
+    - op: add
+      path: /spec/template/spec/containers/0/command/-
+      value: -kube-api-qps=5.0
+    - op: add
+      path: /spec/template/spec/containers/0/command/-
+      value: -kube-api-burst=10
+    - op: remove
+      path: /spec/template/spec/containers/1
```

Comment:
Why can't we do theses(remove external provisioner from controller) changes to our base driver YAML instead of a kustomize layer.

Reply:
We can and I thought I had made that change when squashing. I'll check again.... it looks like I simply forgot to remove this file. It shouldn't be needed anymore.

Initially it wasn't done because distributed and centralized provisioning were both supported for a while.

Reply:
Fixed.